aten/src/ATen/native/cudnn/Conv_v8.cpp

```diff
@@ -359,7 +359,8 @@ auto get_plans_from_find(const cudnnHandle_t handle, const cudnnBackendDescripto
   cudnn_frontend::executionPlans_t valid_plans;
   c10::DeviceGuard g(x.options().device());
   at::DataPtr workspace_ptr;
-  generate_and_filter_plans(handle, opGraph, generator, x, valid_plans, workspace_ptr);
+  auto benchmark_limit = at::globalContext().benchmarkLimitCuDNN();
```

Comment:
how many cudnn specific params do we anticipate? Did the time come to have a special cudnn config in the global context, instead of querying benchmark/deterministic/tf32/benchmarklimit one-by-one? (No action required on this PR, but something to think about for the future)